gem/oq-engine
# EngineWeb: OpenQuake Engine Web Client

First version of web client for oq-engine.
To test it:
 * Setup an environment to run normal engine calculations (hazardlib, risklib, PYTHONPATH)
 * ```cd openquake/server```
 * run:  
```python ./manage.py runserver 0.0.0.0:8000 celery ```
 * Use a browser to connect to ```http://[your_host]:8000/engineweb```
 * At this point upload zip archive of job files and check all cases (good zip, bad zip, good calculation, bad calculation ...)
